static/app/components/links/link.tsx

```diff
@@ -45,7 +45,9 @@ class Link extends React.Component<Props> {
   }
 }
 
-export default withRouter(Link);
+const Link = withRouter(BaseLink);
+
+export default Link;
```

Comment:
I changed this so LSP enabled editors would correctly auto import our Link over react-routers